app/models/ability.rb

```diff
@@ -80,14 +80,13 @@ def initialize(member)
     
     if member.is_at_least? :exec
       # Almost read only finance
-      can :read, :finance
-      can :read, Invoice
-      can :readprice, Invoice
-      can :create, Invoice
-      can :update, Invoice, :status => Invoice::Invoice_Status_Group_Exec
-      can :email, Invoice, :status => Invoice::Invoice_Status_Group_Exec
-      can :read, InvoiceItem
-      can :read, InvoiceContact
+      can :read, :finance_event_url
```

Comment:
Why change this to `finance_event_url`? I believe this will cause the Finance tab to disappear?